src/modules/Dashboard/components/CurrentImageTab/CurrentImageTab.tsx

```diff
@@ -68,77 +61,70 @@ export default class CurrentImageTab extends PureComponent<IProps, IState> {
 			},
 		} = this.props;
 
+		const commonTab = {
+			title: <Trans i18nKey='common.Common'/>,
+			content: (
+				<div className={styles.header}>
+					<img className={styles.header_img} src={thumb} />
+					<div className={styles.header_createdAt}>
+						<span>Created at: </span>
+						<strong className={styles.header_createdAt_date}>21.09.2018</strong>
+					</div>
+				</div>
+			),
+		};
+		const packagesTab = {
+			title: <Trans i18nKey='common.Packages'/>,
+			content: (
+				<div className={styles.dropDownWrapper}>
+					<DropDownTab
+						title={<Trans i18nKey='Packages.Base'/>}
+						value={basePackages.length}
+						isOpened={!!openedDropdownTabs.base}
+						onClick={() => this.onToggleDropdownTabs('base')}
```

Comment:
? use a reference on yor function. 

class ....

myFunc = () => this.onToggleDropdownTabs('base')

onClick={this.myFunc}

Reply:
![Image of Error](https://cdn.discordapp.com/attachments/418795190061629441/545545445477318666/unknown.png)

Reply:
That's your code